goldstone/core/models.py

```diff
@@ -78,10 +77,64 @@ class ReportData(DailyIndexDocType):
 
     INDEX_PREFIX = 'goldstone_reports-'
 
-    class Meta:
+    class Meta:          # pylint: disable=C0111,W0232,C1001
         doc_type = 'core_report'
 
 
+class EventData(DailyIndexDocType):
+    """The model for logstash events data."""
+
+    # The indexes we look for start with this string.
+    INDEX_PREFIX = 'events'
```

Comment:
As this is a prefix, it'll match indexes that start with "events".  At this time, "events" and "events_" will both work the same, if I understand the code correctly.